pkg/config/config_template.yaml

```
@@ -3108,13 +3108,13 @@ api_key:
 
       ## @param cpu - string - optional
       ## @env DD_ADMISSION_CONTROLLER_AUTO_INSTRUMENTATION_INIT_RESOURCES_CPU - string - optional
-      ## Configures the CPU request and limit for the init containers.
+      ## Configures the CPU request for the init containers.
```

maybe we should be more precise and we say:

```suggestion
      ## Configures the CPU request that will be apply for the init container's CPU request and limit.
```